### PR TITLE
15 socioeconomic purpose

### LINKED ISSUE #15: Discussion: Socioeconomic purpose
CoST IDS has an element titled "Purpose" and in draft guidance note 9, this is described as:

> Specify the socio economic purpose of the project

Should we have a field for this in the project level data specification? If so, should it be free text, a document, or using some sort of codelist?

### PR DESCRIPTION
**Pull request by duncandewhurst**
closes #15

### CODE REVIEW

**Comment by jpmckinney** [comment]
I note some trailing text on some rows (below). Can this be removed? 

    `Issue

**Comment by jpmckinney** [comment]
This seems to not be based on recent commits (master split project-level.csv into three CSVs back in December).

**Comment by duncandewhurst** [comment]
Ah, well spotted - this branch pre-dated those changes on master - I've merged the changes into this branch and updated the relevant mapping csv file (and the source spreadsheet).

File: mapping/project-level-identification.csv [hunk 1]
```diff
@@ -4,5 +4,5 @@ Sector,"Develop a list of sectors relevant to country e.g. housing, transport, e
...
 Project Location,Briefly specify location of the project,"Project Level: Publish using the ``locations`` fields as an address, geometry (point/line/polygon) or gazetteer entry","OCDS: Copy from ``planning.project.locations`` (Budget and projects extension, PPP profile); if not available, infer from ``tender.items.deliveryLocation`` or ``tender.items.deliveryAddress`` (Location extension)",
-Purpose,Specify the socio economic purpose of the project,,OCDS: Infer from ``planning.rationale`` and check ``planning.documents`` for documents with ``.type`` set to 'needsAssessment',`Issue #15 <https://github.com/open-contracting/infrastructure/issues/15>`_
+Purpose,Specify the socio economic purpose of the project,"Project Level: Publish a free text description in ``purpose`` and if classified against a local or global codelist then publish in ``additionalClassifications``.",OCDS: Infer from ``planning.rationale`` and check ``planning.documents`` for documents with ``.type`` set to 'needsAssessment',`Issue #15 <https://github.com/open-contracting/infrastructure/issues/15>`_
```
Comment (jpmckinney) [inline mapping/project-level-identification.csv]: Prefer:

> Project Level: Publish in ``purpose``; if classified against a codelist, also publish in ``additionalClassifications``.

"local or global" covers all options, so cut those words for brevity/clarity. We don't have "a free text description" for similar fields; I suggest omitting here as well. Use semi-colon "if" construction as in other mappings, which is easier to read.

**Pull request merged**